.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,19 @@
+name: Node.js CI
+on:
+  push:
+    branches: [ master ]
+  pull_request:
```

Comment:
I'm actually not sure this trailing colon is valid syntax, but we have no way to check until it's merged into the repo because the actions won't run until then.

Reply:
It is. It's from the examples in the github actions documentation.
And I've tested in on my fork, and by loading the file with pyyaml.

What bothers me more is that unquoted `on` is special in yaml (it's an alias for `true` :sweat_smile:), but that's what github uses.

```python
>>> import yaml
>>> yaml.safe_load("on: []")
{True: []}
```